### DISCUSSION TITLE
Exclude In given websites, don't work

### DISCUSSION THREAD

**Discussion by azizsayfiddin** [philc/vimium#4588] [General]
I tried to disable "/" keymap on YouTube, but it doesn't work. Here's what I have in the vimium options
![image](https://github.com/user-attachments/assets/903f518f-5c4a-4435-be2c-bc08e7cfa9c1)

**Comment by philg-dev** [comment]
For me it works perfectly fine to exclude `/` (even on a German keyboard layout, where it's not a single key).

Firefox has a "Quick Search" feature that opens with `/`. It appears at the bottom of the browser window in full width. If you get that, then it's not Vimium. If you get a white search box in the bottom right though, then it's actually Vimium.

Please provide some more information.

**Reply by azizsayfiddin** [reply]
https://github.com/user-attachments/assets/b5ed2356-0618-4f78-a8f7-a1911197b9b4

**Comment by azizsayfiddin** [comment]
It should've been `https?://www.youtube.com/*`
